hosting/src/hooks/useTanamUser.tsx

```diff
@@ -29,6 +29,10 @@ export function useTanamUser(uid?: string): UseTanamDocumentsResult {
     const unsubscribe = onSnapshot(
       docRef,
       (snapshot) => {
+        if (!snapshot.exists()) {
+          setError(new UserNotification("error", "Access Denied", "Sorry you cant access the page"));
+        }
```

Comment:
This is not the way to check it. The issue description is [explicitly requiring](https://github.com/oddbit/tanam/issues/392) you to check the custom claims

<img width="671" alt="image" src="https://github.com/user-attachments/assets/97f11d26-ffb0-462d-b4dc-05c441497ce3">

Reply:
Done ✅